src/createServer.js

```diff
@@ -1,8 +1,66 @@
 'use strict';
 
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const fileName = normalizedURL.pathname.slice(1);
+
+    res.setHeader('Content-Type', 'text/plain');
+
+    if (!req.url.startsWith('/file')) {
+      res.statusCode = 400;
+      res.end('Invalid request');
+
+      return;
+    }
+
+    if (!req.url.startsWith('/file/')) {
+      res.end('Hint message: Use /file/ to access files');
+
+      return;
+    }
```

Comment:
This check for `if (!req.url.startsWith('/file/'))` is valid, but it could be combined with the previous check to streamline the logic. Consider merging these conditions to reduce redundancy.